cross-validation/deepnet/script.whizzml

```diff
@@ -126,6 +129,23 @@
     (filter (lambda (x) (and (pref? x) (pred? x))) (keys fields))))
 
 
+;; check-minority-instances
+;;
+;; Checks whether the number of instances in the minority class is greater
+;; than the number of folds
+(define (check-minority-instances k-folds dataset-id objective-id)
+  (let (dataset (fetch dataset-id)
+        min-cat (last (dataset ["fields" objective-id "summary" "categories"]
+                               []))
```

Comment:
this is going to throw an exception if there are no categories... did you mean:

```
(last (dataset ["fields" ...] []) [])
```

or

```
(last (datasest ["fields" ...] [[]])
```